src/Mod/Sketcher/App/Sketch.cpp

```diff
@@ -5140,17 +5093,17 @@ int Sketch::initBSplinePieceMove(int geoId,
     return 0;
 }
 
-int Sketch::movePoint(int geoId, PointPos pos, Base::Vector3d toPoint, bool relative)
+int Sketch::movePoint(std::vector<std::pair<int, Sketcher::PointPos>> moved,

```

Comment:
I'm not sure that moveByReferencePoint describes the function. Perhaps rather moveGeoElements for the function name. For the variable I think it's overkill since the function already describe. Perhaps moved is not the best though, moving would perhaps be better.

Reply:
Whatever the name, it should not be movePoint, because that can be read as only moving one point. What do you mean by "function only describe".

Moved and moving can both sound like `bool`. You probably want `idsToBeMoved`.